phpBB/phpbb/group/helper.php

```diff
@@ -15,17 +15,58 @@
 
 class helper
 {
-	/** @var  \phpbb\language\language */
+	/** @var \phpbb\auth\auth */
+	protected $auth;
+
+	/** @var \phpbb\cache\service */
+	protected $cache;
+
+	/** @var \phpbb\config\config */
+	protected $config;
+
+	/** @var \phpbb\language\language */
 	protected $language;
 
+	/** @var \phpbb\event\dispatcher_interface */
+	protected $phpbb_dispatcher;
+
+	/** @var \phpbb\path_helper */
+	protected $phpbb_path_helper;
+
+	/** @var \phpbb\user */
+	protected $user;
+
+	/** @var string phpBB root path */
+	protected $phpbb_root_path;
+
+	/** @var string PHP file extension */
+	protected $php_ext;
+
 	/**
 	 * Constructor
 	 *
-	 * @param \phpbb\language\language $language	Language object
+	 * @param \phpbb\auth\auth $auth Authentication object
```

Comment:
Can you please change these to not always use fully qualified class names? (see http://php.net/manual/en/language.namespaces.importing.php)

Reply:
Done
Just for the record, I tried using `use phpbb\cache` and then use `cache\service`, but Travis doesn't like that. See: https://travis-ci.org/phpbb/phpbb/jobs/473299146#L498https://travis-ci.org/phpbb/phpbb/jobs/473299146#L498
Otherwise I had to use `service` which is not very descriptive. So changed it to `use phpbb\cache\service as cache`.